contracts/test/helpers/TestBaseWorkflow.t.sol

```diff
@@ -171,7 +188,47 @@ abstract contract TestBaseWorkflow is DSTest {
       _jbTokenStore,
       _jbSplitsStore
     );
+
+    _jbPaymentTerminalStore = new JBSingleTokenPaymentTerminalStore(
+      _jbDirectory,
+      _jbFundingCycleStore,
+      _jbPrices
+    );
+
+    evm.prank(_multisig);
+    _jbToken = new JBToken('MyToken', 'MT');
```

Comment:
naming is confusing -> _jbToken should be the *project token* while this one should be _randomErc20token, _token, _payMeToken or _safeMoonDogInuERC20